### PR TITLE
added libertyX and Bitaccess

### PR DESCRIPTION
**Pull request by ahujsak**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: copy-plugin.js [hunk 1]
```diff
@@ -10,8 +10,46 @@ const iosDir = './ios/plugins/'
 
 const platforms = [androidDir, iosDir]
 const pluginManifests = {
-  buysell: [],
-  spend: []
+  buysell: [
+    {
+      pluginId: 'co.edgesecure.libertyx',
```
Comment (swansontec) [inline copy-plugin.js]: I would drop the `co.edgesecure` part and just go with the provider's website, like `com.libertyx`.

File: copy-plugin.js [hunk 2]
```diff
@@ -10,8 +10,46 @@ const iosDir = './ios/plugins/'
...
+      pluginURL: 'https://libertyx.com/a/',
+      name: 'libertyX',
+      subtitle: 'Buy bitcoin instantly at trusted stores near you.',
+      provider: 'Edge Wallet',
+      iconUrl: 'https://edge.app/wp-content/uploads/2019/05/libertyXlogo.png',
+      environment: {}
+    },
+    {
+      pluginId: 'co.edgesecure.bitaccess',
```
Comment (swansontec) [inline copy-plugin.js]: `com.bitaccessbtm`

File: copy-plugin.js [hunk 3]
```diff
@@ -10,8 +10,46 @@ const iosDir = './ios/plugins/'
...
+      pluginURL: 'https://bitaccessbtm.com',
+      name: 'Bitaccess',
+      subtitle: 'Buy And Sell Bitcoin Instantly.',
+      provider: 'Edge Wallet',
+      iconUrl: 'https://edge.app/wp-content/uploads/2019/05/bitaccess-symbol.png',
+      environment: {}
+    },
+    {
+      pluginId: 'co.edgesecure.moonpay',
```
Comment (swansontec) [inline copy-plugin.js]: `io.moonpay.buy`

File: copy-plugin.js (hunk 1 above)
Comment (ahujsak) [reply copy-plugin.js]: done

File: copy-plugin.js (hunk 2 above)
Comment (ahujsak) [reply copy-plugin.js]: done

File: copy-plugin.js (hunk 3 above)
Comment (ahujsak) [reply copy-plugin.js]: done

**Pull request merged**